### DISCUSSION TITLE
Xcode 13.1 unstable, keeps crashing

### DISCUSSION THREAD

**Discussion by wogus3602** [pointfreeco/swift-composable-architecture#927] [Q&A]
Xcode 13.1 keeps crashing.

Also in a swift file containing a reducer for TCA.

Does anyone have the same problem as me?

https://developer.apple.com/forums/thread/693396

**Comment by buscarini** [comment]
Same here, I'm not sure how anyone can work with Xcode 13, it's completely unusable for me. I have a project with TCA with ~40 modules, and if I open it I just see a spinning beach ball forever. My only solution so far has been to use Xcode 12 on Monterrey by running the executable directly.
